app/styles/marketing-banner.less

```diff
@@ -0,0 +1,115 @@
+.upf-marketing-banner {
+  --primary-gradiant-color: var(--color-violet-100);
+  --secondary-gradiant-color: var(--color-melon-100);
+  --background-grid-display: flex;
+  --illustration-display: flex;
+
+  container-type: inline-size;
+  container-name: upf-marketing-banner;
+  display: flex;
+  flex: 1;
+  background-color: var(--color-white);
+  border-radius: var(--border-radius-md);
+  box-shadow: var(--box-shadow-sm);
+  overflow: hidden;
+
+  &__container {
+    position: relative;
+    display: flex;
+    flex: 1;
+
+    padding: var(--spacing-px-18);
+    gap: var(--spacing-px-18);
+
+    &:before {
+      display: var(--background-grid-display);
+      content: '';
+      top: 0;
+      left: 0;
+      width: 270px;
+      height: 150px;
+      position: absolute;
+      background-image: url('/@upfluence/oss-components/assets/components/marketing-banner/grid.svg');
+    }
+  }
+
+  &__gradient--primary,
+  &__gradient--secondary,
+  &__gradient--tertiary {
```

Comment:
I don't understand how these are gradients 🤔 
Shouldn't they have some sort of `linear-gradient` css property ?

Reply:
I can rename it. In the first implementation, it was gradient but I change that after a sync with Hsinyi